src/bgw/job.c

```diff
@@ -377,7 +377,7 @@ ts_bgw_job_get_scheduled(size_t alloc_size, MemoryContext mctx)
 		 * handle them below. We can only use memcpy for the non-nullable fixed
 		 * width starting part of the BgwJob struct.
 		 */
-		memcpy(job, GETSTRUCT(tuple), offsetof(FormData_bgw_job, fixed_schedule));
+		memcpy(job, GETSTRUCT(tuple), offsetof(FormData_bgw_job, initial_start));
```

Comment:
The `fixed_schedule` field is now included and all fields starting with `initial_start` are initialized below.